plugins/DataLoadCSV/dataload_csv.cpp

```diff
@@ -224,6 +228,14 @@ bool DataLoadCSV::readDataFromFile(FileLoadInfo* info, PlotDataMapRef& plot_data
     plot_data.numeric.clear();
   }
 
+  if (ignored_lines > 0 && lines_read > 0)
+  {
+    QString message = QString("Ignored %1 out of %2 lines read from file\n"
+                              "Valid delimiters are %3"
+                             ).arg(ignored_lines).arg(lines_read).arg(csv_separator.pattern());
```

Comment:
The delimiters will be shown in regex form but may want to show in a more user friendly form.